.vib/gotrue/goss/vars.yaml

```diff
@@ -0,0 +1,6 @@
+binaries:
+  - gotrue
+files:
+  - paths:
+      - /opt/bitnami/gotrue/schema.sql
+root_dir: /opt/bitnami
```

Comment:
As it is not possible to obtain the `gotrue` version, I would suggest implementing a `gotrue help` test to ensure the main binary is executable.

Reply:
Right, the `gotrue version` command returns a commit id, we cannot test it against `APP_VERSION`. Should we add the help test even if we know the binary is executable because it is ran in the supabase chart by default?

Reply:
I would say so. Although downstream the `bitnami/chart` test will run the `gotrue` binary and will test more extensively its functionalities, testing here a `--version` test (or, in its absence, a `--help` test) allows us to asses that the binary is present and is executable and does not return an error before releasing the image.

Although, I would suggest not setting a `stdout` condition with many precise matches to avoid errors caused by intended changes in upstream.